cmd/cli/main.go

```diff
@@ -1,38 +0,0 @@
-package main
-
-import (
-	"flag"
-	"log"
-
-	"notion-blog/internal"
-	notion_blog "notion-blog/pkg"
-
-	"github.com/itzg/go-flagsfiller"
-	"github.com/joho/godotenv"
-)
-
-var config notion_blog.BlogConfig
-
-func parseFlagsConfig() {
-	// create a FlagSetFiller
-	filler := flagsfiller.New()
-	// fill and map struct fields to flags
-	err := filler.Fill(flag.CommandLine, &config)
```

Comment:
This is quite crucial to automatically generate flags according to the config struct.